documentation/add/07-sqw_redesign.md

```diff
@@ -65,7 +91,7 @@ This data should all be available from the Mantid `.nxspe` file, however it is l
 (1): the format of this datafile is TBD. To ease the eventual integration with Mantid a Nexus/HDF5 file or some other structured data that maps easily into the HDF5 format should be used.
 
 ####  Instrument specification (`IX_instr`)
-The instrument class contains the information about the components that make up the instrument pre-sample, including the choppers, moderators and incident beams. 
+The instrument class contains the information about the components that make up the instrument pre-sample, including the choppers, moderators and incident beams.
```

Comment:
IX_instr->IX_inst and its subclasses.
Below, L.96 change 'the object contains' to 'The specific subclasses contain'

Reply:
hopefully modified accordingly